frontend/tablet/src/screens/RoomDetailScreen/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { StyleSheet, View } from 'react-native'
+
+import { Typography, Button } from '../../components'
+import theme from '../../components/theme'
+
+const Footer: React.FC = () => {
+  return (
+    <>
+      {/* TODO extract style */}
+      <View style={styles.quickReservation}>
+        <Typography variant="button">QUICK RESERVATION</Typography>
+        {/* TODO extract style */}
+        <View style={styles.buttons}>
+          <Button
+            title="15 min"
+            // eslint-disable-next-line no-console
+            onPress={() => console.log('TODO reserve 15 min')}
+            margin
+          />
+          <Button
+            title="30 min"
+            // eslint-disable-next-line no-console
+            onPress={() => console.log('TODO reserve 30 min')}
+            margin
+          />
+          <Button
+            title="45 min"
+            // eslint-disable-next-line no-console
+            onPress={() => console.log('TODO reserve 45 min')}
+            margin
+          />
+          <Button
+            title="60 min"
+            // eslint-disable-next-line no-console
+            onPress={() => console.log('TODO reserve 60 min')}
+          />
+        </View>
+      </View>
+
+      {/* TODO extract style */}
+      <View style={styles.nextMeeting}>
+        <Typography variant="button">NEXT MEETING</Typography>
+        <Typography variant="h4" style={styles.meetingName}>
+          Team Meeting
+        </Typography>
+        <Typography>14:30</Typography>
+        <Typography color="gray">Karol Sloboda</Typography>
+      </View>
+    </>
+  )
+}
+
+export default Footer
+
+const styles = StyleSheet.create({
+  quickReservation: {
+    justifyContent: 'flex-end',
+  },
+  buttons: {
+    flexDirection: 'row',
+    marginTop: theme.spacing.md,
```

Comment:
Let's make it `xl`, looks better IMO, what do you think?

Reply:
Yes I agree, it can be on `xl` size.